src/utils/api.ts

```diff
@@ -41,6 +41,7 @@ export const api = createTRPCNext<AppRouter>({
         }),
         httpBatchLink({
           url: `${getBaseUrl()}/api/trpc`,
+          maxURLLength: 2083,
```

Comment:
https://trpc.io/docs/client/links/httpBatchLink#setting-a-maximum-url-length

This was an error occurring in the website.

Reply:
👍